src/components/LandingPage.js

```diff
@@ -3,47 +3,59 @@ import {connect} from 'react-redux';
 import {Hero, Section} from 'react-landing-page';
 import {GoogleLogin} from 'react-google-login';
 import Cookies from 'js-cookie'
-import {loginBackend} from "../backend/yaas";
-import {setUserSignedInAction, ACCESS_TOKEN_COOKIE_KEY} from '../reducers/UserSignedIn';
+import {
+  BACKEND_MODE_SIGNED_IN_STATUS,
+  ACCESS_TOKEN_COOKIE_KEY,
+  loginBackend,
+  setBackendModeSignedInStatusAction,
+} from '../reducers/BackendModeSignedInStatus';
 
 const CLIENT_ID = '709358329925-gic89ini15sgaenfrta1gshej1ik72jg.apps.googleusercontent.com';
 
 class LandingPage extends React.Component {
 
-	handleLoginSuccess = response => {
-		if (response.tokenObj) {
-			loginBackend(response.profileObj.name, response.profileObj.email, response.tokenObj.id_token).then(ok => {
-				if (ok) {
-					Cookies.set(
-						ACCESS_TOKEN_COOKIE_KEY,
-						response.tokenObj.id_token,
-						{ expires: new Date(response.tokenObj.expires_at) },
-					);
-					this.props.dispatchSetUserSignedInAction(true);
-				}
-			});
-		}
-	};
+  handleLoginSuccess = response => {
+    if (response.tokenObj) {
+      loginBackend(response.profileObj.name, response.profileObj.email, response.tokenObj.id_token).then(ok => {
+        if (ok) {
+          Cookies.set(
+            ACCESS_TOKEN_COOKIE_KEY,
+            response.tokenObj.id_token,
+            { expires: new Date(response.tokenObj.expires_at) },
+          );
+          this.props.dispatchSetBackendModeSignedInStatusAction(BACKEND_MODE_SIGNED_IN_STATUS.USER_SIGNED_IN);
+        }
+      });
+    }
+  };
 	
-	render = () =>
-		<Hero
-			color="black"
-			bg="white"
-			backgroundImage="https://source.unsplash.com/npxXWgQ33ZQ/1600x900">
-			<Section heading='YADA' subhead='Yet Another Docs App' width={1}>
-				<GoogleLogin
-					clientId={CLIENT_ID}
-					buttonText="Sign in with Google to get started"
-					onSuccess={this.handleLoginSuccess}
-					onFailure={response => { console.log(response); }}
-					cookiePolicy="single_host_origin"
-					responseType="code,token"
-				/>
-			</Section>
-		</Hero>;
+  render = () =>
+    <Hero
+      color="black"
+      bg="white"
+      backgroundImage="https://source.unsplash.com/npxXWgQ33ZQ/1600x900">
+      <Section heading='YADA' subhead='Yet Another Docs App' width={1}>
+        <GoogleLogin
+          clientId={CLIENT_ID}
+          buttonText="Sign in with Google to get started"
+          onSuccess={this.handleLoginSuccess}
+          onFailure={response => { console.log(response); }}
+          cookiePolicy="single_host_origin"
+          responseType="code,token"
+        />
+      </Section>
+      <button
+        onClick={() => {
+          this.props.dispatchSetBackendModeSignedInStatusAction(BACKEND_MODE_SIGNED_IN_STATUS.LOCAL_STORAGE);
+        }}>
+        Use local storage
+      </button>
```

Comment:
I added this ugly looking button here for now just to switch, but we should probably remove it later.

Also Idk whether or not we should try and keep the local storage backed version (probably not once we have to deal with editing views since we need the graph stuff to be able to access the tagged blocks properly - I was previously experimenting with storing some extra stuff to be able to do a localstorage backed version, but it gets pretty complicated). 

Anyways, I kept the local storage stuff in the FIleStorageSystem module for now, but we can remove later as well.

Reply:
Oh didn't see this, so my previous comment is kind of invalid if we do decide to remove the local storage cache. Could be cool to keep it, but I'm not sure what the effort will be.